### PR TITLE
Modify definitions to translate content block

### PR DESCRIPTION
**Pull request by MebinAbraham**
Needed to the updated translation of definitions since the structure was changed for: https://trello.com/c/rC82gUmp/2180-move-significant-changes-to-use-definition-pattern

Make sure translation still works as expected.

Related PRs:
1. https://github.com/ONSdigital/eq-survey-runner/pull/1712
2. https://github.com/ONSdigital/eq-schema-validator/pull/72

### CODE REVIEW

**Comment by benhowes** [comment]
@MebinAbraham depending on our discussion over at https://github.com/ONSdigital/eq-survey-runner/pull/1712, it might make sense to also test translating the `list`/`title`

File: app/translate_survey.py [hunk 1]
```diff
@@ -258,8 +258,14 @@ def translate_introduction_content(container, context, translations):
 
 def translate_definitions_text(container, context, translations):
     if 'definitions' in container:
-        for definitions in container['definitions']:
-            translate_container(definitions, context, translations)
+        for definition in container['definitions']:
+            translate_container(definition, context, translations)
+
+            for item in definition['content']:
```
Comment (jonnyshaw89) [inline app/translate_survey.py]: Maybe we should define a common function to extract from a "content" type? seeing as we use the pattern in multiple places?

**Comment by MebinAbraham** [comment]
@jonnyshaw89 Re-review please.

File: app/translate_survey.py [hunk 2]
```diff
@@ -263,10 +263,12 @@ def translate_calculated_summary(container, translations):
...
         translate_titles_text(container['calculation'], container['id'], translations)
 
+
 def translate_definitions_text(container, context, translations):
     if 'definitions' in container:
-        for definitions in container['definitions']:
-            translate_container(definitions, context, translations)
+        for definition in container['definitions']:
+            translate_container(definition, context, translations)
+            translate_block_content(definition, context, translations)
```
Comment (jonnyshaw89) [inline app/translate_survey.py]: should this rename to `translate_content_block` maybe? as its not the `block` that this is now just used for?

Comment (MebinAbraham) [reply app/translate_survey.py]: @jonnyshaw89 Sorted.

**Pull request merged**